app/models/saved_claim/dependency_claim.rb

```diff
@@ -106,9 +106,9 @@ def submittable_686?
   end
 
   def submittable_674?
-    return false if parsed_form['view:selectable686_options']['report674'].blank?
-
-    true
+    # check if report674 is present and then check if it's true to avoid hash break.
+    parsed_form['view:selectable686_options']&.include?('report674') &&
+      parsed_form['view:selectable686_options']['report674']
```

Comment:
Would dig work better?
```suggestion
parsed_form.dig('view:selectable686_options', 'report674') == true
```